tests/layer_tests/jax_tests/jax_layer_test_class.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import itertools
+from copy import deepcopy
+
+import numpy as np
+from common.constants import test_device, test_precision
+from jax import numpy as jnp
+from openvino.runtime import Core
+
+
+class JaxLayerTest:
+    def _test(self, model, ref_net, ie_device, precision, ir_version, infer_timeout=60, dynamic_shapes=True,
+              **kwargs):
+        """
+        :param enabled_transforms/disabled_transforms: string with idxs of transforms that should be enabled/disabled.
```

Comment:
Do we need this? It looks like wrong regarding list of parameters. Please, check

Reply:
no, we don't. Let me remove this one in the next PR